08-mapping.Rmd

```diff
@@ -64,14 +64,13 @@ The generic `plot()` function is often the fastest way to create static maps fro
 Sometimes simplicity and speed are priorities, especially during the development phase of a project, and this is where `plot()` excels.
 The base R approach is also extensible, with `plot()` offering dozens of arguments.
 Another low-level approach is the **grid** package, which provides functions for low-level control of graphical outputs, --- see *R Graphics* [@murrell_r_2016], especially  Chapter [14](https://www.stat.auckland.ac.nz/~paul/RG2e/chapter14.html).
-The focus of this section, however, is making static maps with **tmap**.
+The focus of this section, however, is making static maps with **tmap**, emphasizing the important aesthetic and layout options.
```

Comment:
True that. The sentence now gives a more accurate account of what is coming in the rest of the section.

Reply:
I was mostly trying to get rid of the repeated sentence. Glad you were ok with that :slightly_smiling_face: (I have otherwise been trying to limit myself to typos and actual mistakes, rather than style changes).